dsn-2018/bare_conf.tex

```diff
@@ -0,0 +1,1264 @@
+\end{align}
+This is convenient for when the specifics of the differential path do not
+matter, merely that there exist at least one collision class of the specified
+form.
+
+
+\subsection{Neighborhoods}
+
+We define the neighborhood of a collision class, $C \in \mathcal{C}$, to be the
+set of all other non-empty collision classes at a fixed distance,
+$d \in \mathbb{N}$, from $C$. That is:
+\begin{align}
+    N(C, d) & = \{ C_j \in \mathcal{C} : \delta(C, C_j) = d \} \\
+    N(C) & = N(C, 1)
+\end{align}
+For instance, $C_{Wang} \in N(C_{Sasaki}, 21)$. The distance parameter, $d$,
+may optionally be omitted, in which case the unit distance is implied. Thus,
+$C_{Kasselman} \in N(C_{Dobbertin})$.
+
+We can similarly define the neighborhood of a family of collision classes,
+$F \in \mathcal{F}$, to be the set of all other families of collision classes
+at a fixed distance, $d \in \mathbb{N}$, from $F$. That is:
+\begin{align}
+    \mathcal{N}(F, d) & = \{ F_j \in \mathcal{F} : \delta(F, F_j) = d \} \\
+    \mathcal{N}(F) & = N(F, 1)
+\end{align}
+The distance parameter, $d$, may optionally be omitted, in which case the unit
+distance is implied.
+
+Neighborhoods can be classified into three types: \textit{expansion},
+\textit{internal}, and \textit{mixed}. Let $d$ be fixed. An \textit{expansion}
+neighborhood of a collision class, $C \in \mathcal{C}$, is the neighborhood
+restricted only to those collision classes which only differ in rounds external
+to the collision family of $C$. An \textit{internal} neighborhood of $C$ is the
+neighborhood restricted only to those collision classes which differ in rounds
+internal to the collision family of $C$. An \textit{mixed} neighborhood of $C$
+is the neighborhood restricted only to collisions which differ in a round
+internal and a round external to the collision family of $C$. That is:
+\begin{align*}
+    N_{exp}(C, d) = \{ & C_{j} \in N(C, d) : \forall i \in F(C), C_j[i] = C[i] \} \\
+    N_{int}(C, d) = \{ & C_{j} \in N(C, d) : \forall i \not\in F(C), C_j[i] = C[i] \} \\
+    N_{mix}(C, d) = \{ & C_{j} \in N(C, d) : \exists i \in F(C), C_j[i] \neq C[i] \\
+        & \text{ and } \exists k \not\in F(C), C_j[k] \neq C[k] \}
+\end{align*}
+
+
+
+\section{Intuition on the Utility of a Collision} \label{Sec:Intuition}
+
+\textbf{TODO - audit introductory paragraphs}
+
+We seek to build new intuition about collisions in hash functions
+and particularly collisions in MD4. By developing techniques using logical
+cryptanalysis, we seek to complement and extend the results from
+differential cryptanalysis. In particular, we wish to use logical cryptanalysis
+to discuss the impacts of collisions in real-world systems. In this paper,
+we introduce and give justification for general techniques, and present some
+results of these techniques.
+
+To begin, real world systems are most vulnerable to two attacks: the preimage
+and second preimage. For systems relying on e.g., password validation via
```

Comment:
The use of "e.g" here feels unnecessary and since the next sentence follows a similar format without "e.g." makes it feel unbalanced. But that might just be me not understanding what you're trying to convey.